### PR TITLE
Add notes to QA changes to extension

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: browser/README.md [hunk 1]
```diff
@@ -18,20 +15,51 @@ $ make release
...
 
+## Manual Install and QA
 
-## Manual Install
+Build a fresh extension to load into your browsers:
 
-### Chrome Extension
+
+```shell
+$ yarn; make release
```
Comment (shazow) [inline browser/README.md]: Can you confirm that this actually works? :D I forget if there's a bit more setup required upfront.

Comment (joshblum) [reply browser/README.md]: works!

File: browser/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Build a fresh extension to load into your browsers:
 
 
 ```shell
-$ yarn; make release
+$ yarn; make bundle
```
Comment (shazow) [inline browser/README.md]:
```shell
$ yarn
$ make
```

should do the trick

Comment (joshblum) [reply browser/README.md]: fixed

**Pull request merged**